translations/es/README.md

```diff
@@ -0,0 +1,238 @@
+
+1. API's CRUD (crear, leer, actualizar, eliminar (por sus siglas en inglés)). Para leer y manipular modelos.
+
+2. API's analíticas para mediciones de agregación sobre uno o mas atributos del modelo.
+
+Elide soporta un número de características:
+
+### Seguridad se convierte en estándar
+Control de acceso a campos y entidades a trvés de una sintaxis de permisos declarativa e intuitiva.
+
+### API's amigables con el móvil (Mobile friendly)
+
+JSON-API y GraphQL permite a desarrolladores solicitar grafos de objetos enteros en una simple petición. Solo los elementos solicitados del modelo de datos son devueltos.
+Nuestro enfoque obstinado para mutaciones toma en cuenta escenarios comúnes de aplicaciones como:
+* Crear un nuevo objeto y agregarlo a una colección existente en una misma operación.
+* Crear un conjunto de objetos relacionados (un subgrafo) y conectarlo a un grafo existente.
+* Diferenciar entre borrar un objeto y desasociar un objeto de una relación (sin borrarlo).
+* Referenciar un objeto recién creado dentro de otras operaciones de mutación.
+
+Filtrado, ordenado, paginación y búsqueda de texto tienen soporte fuera de la caja.
+
+### Atomicidad para escrituras complejas
+Elide soporta múltiples mutaciones de modelos de datos en una sola petición, ya sea en JSON-API o GrapQL. Crear objetos, agregarlos a relaciones, modificar o eliminar, todo junto en una sola petición atómica.
+
+### Soporte de consulta analítica
+Elide soporta consultas analíticas en los modelos creados gracias a su poderosa capa de semántica. API´s de Elide trabaja de forma nativa con [Yavin](https://github.com/yahoo/yavin) para visualizar, explorar y reportar en sus datos.
+
+### Introspección de esquema
+Explore, entienda y componga consultas en nuestra API Elide a través de documentación generada por Swagger o por el esquema de GraphQL.
+
+### Personalice
+Personalice el comportamiento de las operaciones del modelo de datos con atributos computados, anotaciones de validación de datos y solicite hooks de ciclo de vida.
+
+### Almacenamiento agnóstico
+Elide es agnóstico a su strategia de persistencia particular. Use un ORM o su propia implementación de almacén de datos.
+
+## Documentación
+
+Mas información acerca de Elide en [elide.io](https://elide.io).
+
+## Instalación
+
+Para probar un ejemplo de servicio de Elide, revise este [Projecto de ejemplo en Spring Boot](https://github.com/yahoo/elide-spring-boot-example).
+
+Alternativamente use [elide-standalone](https://github.com/yahoo/elide/tree/master/elide-standalone) que permite una configuración rápida de una instancia local de Elide ejecutándose de manera embebida en una aplicación Jetty.
+
+## Uso
+
+Los siguientes ejemplos aprovechan elide 5 (ahora en pre-lanzamiento). Para documentación en versiones anteriores/estables, visite [aquí](https://elide.io/pages/guide/v4/01-start.html).
```

Comment:
I forgot to remove this blurb after Elide 5 release.  We can remove the following text altogether:

"Los siguientes ejemplos aprovechan elide 5 (ahora en pre-lanzamiento). Para documentación en versiones anteriores/estables, visite [aquí](https://elide.io/pages/guide/v4/01-start.html)."

Reply:
ok will remove it and start working on the CONTRIBUTING.md file as well